docs/api/schema.yml

```diff
@@ -1821,6 +2006,9 @@ components:
           type: string
         repo_name:
           type: string
+        has_truncated_issues:
+          type: boolean
+          readOnly: true
```

Comment:
@jgerigmeyer @oluoluoxenfree I added a new boolean field has_truncated_issues to the Project. Should be able to tell us if the repo has more than 1000 issues